Source/LightboxController.swift

```diff
@@ -271,7 +271,7 @@ open class LightboxController: UIViewController {
 
   // MARK: - Layout
 
-  open func configureLayout(_ size: CGSize = UIApplication.shared.delegate?.window??.bounds.size ?? .zero) {
+  open func configureLayout(_ size: CGSize) {
     scrollView.frame.size = size
```

Comment:
We always pass `self.view.bounds.size` into `configureLayout`, then why do we need this parameter 🤔 @vadymmarkov @zenangst

Reply:
I'm not sure why we have it, so I'm fine with removing this default value.

Reply:
@andreyrd I think we can just use the default view size inside `configureLayout` without passing it as a parameter, mind making the change 😉

Reply:
It looks like a value other than `view.bounds.size` is passed into `configureLayout` in `viewWillTransition`. (Should be line 220)